sufia-models/app/jobs/create_derivatives_job.rb

```diff
@@ -8,7 +8,11 @@ def run
     if generic_file.video?
       return unless Sufia.config.enable_ffmpeg
     end
-    generic_file.create_derivatives
+    status = Timeout::timeout(Sufia.config.derivatives_timeout) do
```

Comment:
We are using the latest hydra derivatives and sufia and still seeing jobs lasting forever.  It is causing the entire system to seem like it is broken.  I want to have some control over something that is causing the system to break.